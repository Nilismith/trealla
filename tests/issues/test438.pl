:- initialization(main).

main :-
	call(user:((fail,1),_=[])).

