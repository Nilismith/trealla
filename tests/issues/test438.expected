   Warning: Initialization goal exception: error(type_error(callable,((fail,1),_3=[])),call/1).
